sdks/wasm/tests/browser/WebSocketTestSuite.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using WebAssembly.Net.Http.HttpClient; 
+using System.Threading.Tasks;
+using WebAssembly;
+using WebAssembly.Core;
+using System.Threading;
+using System.IO;
+using ClientWebSocket = WebAssembly.Net.WebSockets.ClientWebSocket;
+using System.Net.WebSockets;
+using System.Text;
+
+namespace TestSuite
+{
+    public class Program
+    {
+		static ClientWebSocket CreateWebSocket (Uri server, string protocols = "")
+		{
+			var cws = new ClientWebSocket ();
+
+			if (!string.IsNullOrEmpty(protocols)) {
+
+				foreach (var p in protocols.Split(';')) {
+					cws.Options.AddSubProtocol (p);
+				}
+
+			}
+
+			return cws;
+
+		}
+		public async Task<int> ConnectWebSocketStatus (Uri server, string protocols)
+		{
+			var cws = CreateWebSocket (server, protocols);
+
+            WebSocketCloseStatus status = cws.CloseStatus ?? WebSocketCloseStatus.Empty;
```

Comment:
indentation damage all over in this file, fyi

Reply:
should be fixed in recent commit.  Totally forgot to check format.